en/docs/setup/setting-up-separate-databases-for-clustering.md

```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
-    
+    !!! note
+        You can use the embedded H2 database in development environments. For more information on
+        databases, see [Working with Databases](../../administer/working-with-databases/).
 
-You can create the following databases and associated datasources. This
-is **NOT** mandatory and you can choose to not create these databases if
-you wish and simply have a single database to handle all these concerns.
+You can create the following databases logical separation of data. This
+is **NOT mandatory** and you can choose not to create these databases if
+you wish have default databases to handle all these concerns.
 
 <table>
 <thead>
 <tr class="header">
 <th>Database Name</th>
 <th>Description</th>
-<th>Script location</th>
-<th>Datasource file to be modified</th>
 </tr>
 </thead>
 <tbody>
 <tr class="odd">
 <td><code>             USERSTORE_DB            </code></td>
-<td>Authorization manager configurations, internal permissions and roles.</td>
-<td><pre><code>&lt;IS_HOME&gt;/dbscripts/</code></pre></td>
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/master-datasources.xml            </code></td>
+<td>
+<li>Contains authorization manager configurations, internal permissions and roles.</li>
+<li>In the default database configuration, user data and registry data are located inside <code>SHARED_DB</code>
+.</li>
+<li>For more information, see <a href="../../administer/working-with-databases">Changing the Default Datasource for 
+User Database</a>.</li>
+</td>
 </tr>
 <tr class="even">
 <td><code>             IDENTITY_DB            </code></td>
-<td><p>Contains identity related data. For example, OAuth 2.0, SAML 2.0, user managed access ( UMA) etc.</p></td>
-<td><div class="content-wrapper">
-<pre><code>&lt;IS_HOME&gt;/dbscripts/identity/</code></pre>
-<p>
-<div class="admonition tip">
-<p class="admonition-title">Tip</p>
-<p>UMA related scripts are in the <code><IS_HOME>/dbscripts/identity/uma</code> directory. You can run the scripts to create the UMA related tables and indexes in the <code>IDENTITY_DB</code>.</p>
-</div></p>
-</div></td>
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/master-datasources.xml            </code></td>
-</tr>
-<tr class="odd">
-<td><code>             METRICS_DB            </code></td>
-<td>Metrics Database is used to store the runtime metrics data reported by the Metrics JDBC Reporter periodically. There are 5 tables to store metrics data for each metric type. The metric types are Counter, Meter, Gauge, Histogram, and Timer.</td>
-<td><pre><code>&lt;IS_HOME&gt;/dbscripts/</code></pre></td>
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/metrics-datasources.xml            </code></td>
+<td>
+<li>Contains identity related data. For example, OAuth 2.0, SAML 2.0, User Managed Access (UMA) etc.</li>
+<li>In the default database configuration, all the UMA, concent and identity data are located <code>IDENTITY_DB</code>.
```

Comment:
are located in the

Reply:
Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c